### ISSUE TITLE
Error extracting title when downloading playlists from Youtube.

### ISSUE THREAD

**Issue by 0x27** [rg3/youtube-dl#5040]
Here is the trace:

```
[balor@magh-meall Music]$ youtube-dl --verbose "https://www.youtube.com/watch?v=6-wEAeNcA_A&list=UL6-wEAeNcA_A"
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'https://www.youtube.com/watch?v=6-wEAeNcA_A&list=UL6-wEAeNcA_A']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.23.1
[debug] Python version 2.7.9 - Linux-3.18.6-1-ARCH-x86_64-with-glibc2.2.5
[debug] exe versions: ffmpeg 2.5.4, ffprobe 2.5.4, rtmpdump 2.4
[debug] Proxy map: {}
[youtube:playlist] Downloading playlist UL6-wEAeNcA_A - add --no-playlist to just download video 6-wEAeNcA_A
[youtube:playlist] UL6-wEAeNcA_A: Downloading webpage
ERROR: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/youtube_dl/YoutubeDL.py", line 643, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 275, in extract
    return self._real_extract(url)
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/youtube.py", line 1325, in _real_extract
    page, 'title')
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 557, in _html_search_regex
    res = self._search_regex(pattern, string, name, default, fatal, flags, group)
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 547, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

[balor@magh-meall Music]$ 
```

It appears that something is breaking in the regexes used to extract the videos title from the page. I am unsure where to start digging into it, but guess it might be to do with perhaps a minute change in Youtube's user interface/HTML code?

Interestingly, non playlist downloads still work. When I remove the playlist bit of the URL, it all works fine.

**Comment by dstftw** [comment]
Could you post `*.dump` files you get when running with `--write-pages` option?

**Comment by 0x27** [comment]
Sure, link to the dump file I got is the following: https://transfer.sh/uBq7N/out.dump

Trace also included for interest and incase it helps.
```
[balor@magh-meall Music]$ youtube-dl --verbose --write-pages "https://www.youtube.com/watch?v=6-wEAeNcA_A&list=UL6-wEAeNcA_A"
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', '--write-pages', 'https://www.youtube.com/watch?v=6-wEAeNcA_A&list=UL6-wEAeNcA_A']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.23.1
[debug] Python version 2.7.9 - Linux-3.18.6-1-ARCH-x86_64-with-glibc2.2.5
[debug] exe versions: ffmpeg 2.5.4, ffprobe 2.5.4, rtmpdump 2.4
[debug] Proxy map: {}
[youtube:playlist] Downloading playlist UL6-wEAeNcA_A - add --no-playlist to just download video 6-wEAeNcA_A
[youtube:playlist] UL6-wEAeNcA_A: Downloading webpage
[youtube:playlist] Saving request to UL6-wEAeNcA_A_https_-_www.youtube.com_playlistlist=UL6-wEAeNcA_A.dump
ERROR: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/youtube_dl/YoutubeDL.py", line 643, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 275, in extract
    return self._real_extract(url)
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/youtube.py", line 1325, in _real_extract
    page, 'title')
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 557, in _html_search_regex
    res = self._search_regex(pattern, string, name, default, fatal, flags, group)
  File "/usr/lib/python2.7/site-packages/youtube_dl/extractor/common.py", line 547, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

[balor@magh-meall Music]$ ls
Senbonzakura - cover by Lindsey Stirling-6-wEAeNcA_A.mp3  UL6-wEAeNcA_A_https_-_www.youtube.com_playlistlist=UL6-wEAeNcA_A.dump
[balor@magh-meall Music]$ curl --upload-file UL6-wEAeNcA_A_https_-_www.youtube.com_playlistlist\=UL6-wEAeNcA_A.dump https://transfer.sh/out.dump
https://transfer.sh/uBq7N/out.dump
[balor@magh-meall Music]$ 
```

**Comment by dstftw** [comment]
This will be fixed in the next version. Thanks for the report.

**Comment by 0x27** [comment]
Awesome, thanks :)

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl version 2015.02.24 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.

**Comment by 0x27** [comment]
Seems to be fixed, grabbed latest version via pip :)
